libsentrykube/utils.py

```diff
@@ -290,57 +291,64 @@ def ensure_libsentrykube_folder() -> Path:
 
 
 @cache
-def ensure_kubectl(version: str = KUBECTL_VERSION) -> Path:
+def ensure_kubectl(
+    binary: str = KUBECTL_BINARY, version: str = KUBECTL_VERSION
+) -> Path:
     base = ensure_libsentrykube_folder() / "kubectl" / f"v{version}"
-    path = base / "kubectl"
+    path = base / binary
     if path.is_file():
         return path
 
-    base.mkdir(parents=True, exist_ok=True)
+    if binary == "kubectl":
+        base.mkdir(parents=True, exist_ok=True)
 
-    import hashlib
-    import platform
+        import hashlib
+        import platform
 
-    import click
-    import httpx
+        import click
+        import httpx
```

Comment:
I would recommend to move these imports to the top of the file to align with python best practices.